src/environment.go

```diff
@@ -501,3 +502,19 @@ func cleanHostName(hostName string) string {
 	}
 	return hostName
 }
+
+func returnOrBuildCachePath(path string) string {
+	// validate root path
+	if _, err := os.Stat(path); err != nil {
+		return ""
+	}
+	// validate oh-my-posh folder, if non existent, create it
```

Comment:
@ralish it will always create the oh-my-posh folder inside the cache directory. Meaning that we will write to for example `$env:XDG_CACHE_PATH/oh-my-posh/omp.cache` in this specific use-case.